_posts/2024-07-24-welcoming-resident-fellow-seun-olufemi.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Welcoming OLS Resident Fellow Seun Olufemi to OLS-9 Delivery Team
+authors:
+- malvikasharan
+- seunolufemi123
+image: https://images.unsplash.com/photo-1554457606-ed16c39db884
+photos:
+  name: David Rotimi
+  license: CC BY-SA 4.0
+  url: https://unsplash.com/photos/white-arc-LxENUKJXh_k
+---
+
+We recently announced the [ninth cohort of Open Seeds - OLS-9]({% link _posts/2024-07-15-openseeds-in-2024.md %}), our open science training and mentoring program. As part of this cohort, we have created a **Resident Fellow role**, which was announced with an invitation for applications from our facilitator community.
+
+We are grateful to all our applicants who presented strong project plans and shared their vision for this role. Since we could only offer one position, the decision was based on the applicant's project goals and the feasibility of the OLS team to support them over the next eight months as part of the Open Seeds OLS-9 delivery plans.
+
+**We are delighted to announce that Seun has been offered this role of the OLS Resident Fellow!**
+Seun will work with Malvika Sharan and Tajuddeen (Taj) Gwadabe as part of the OLS-9 delivery team.
+
+<div class="columns">
+  <div class="column is-4" markdown="1">
+  ![A photo of Seun, a man from Nigeria. He is wearing a smile and a colourful, bright African round-neck shirt.](https://hackmd.io/_uploads/B1EogG_uA.jpg)
+  [add links to different orgs above]
```

Comment:
@malvikasharan Are you happy with the links you've added? 😀

I'd like to delete your reminder before we go live.

Reply:
Looks great! Thank you Debs.

Reply:
Where are the links?

Reply:
I believe the links are the ones below 👇🏽

Meaning, we can remove this line.

Reply:
yes, please delete that - we needed to add link to the OLS-9 Blog.

Reply:
Deleted already. 😃